web/src/lib/types/types.ts

```diff
@@ -353,7 +353,7 @@ export const owningFlowDisplayFields = Object.keys(emptyOwningFlowContent) as (k
 
 export const startFlowDisplayFields = Object.keys(emptyStartingFlowContent) as (keyof StartingFlowContent)[];
 
-const businessUserDisplayFields = Object.keys(emptyBusinessUser) as (keyof BusinessUser)[];
```

Comment:
I think this will mess up typing in our IDE's, no?

Reply:
Will it? I'm not sure. I don't **think** it changed anything for me?